config/routes.rb

```diff
@@ -10,6 +10,8 @@
     post 'auth/callback'
     get  'auth/sign_out'
 
+    resources :user, only: %i[index update destroy], path: 'profile'
```

Comment:
I think this (and corresponding controller actions + views) should be changed to a *singular* `profile` resource, since it always acts on a single user. It's possible in the future that we do want to introduce a public `users` list so we should keep that separate. So in concrete terms, the route would be:

```ruby
resource :profile, only: %i[show update destroy]
```

Reply:
This has been updated